covfee/client/tasks/continuous_annotation/index.tsx

```diff
@@ -307,14 +309,23 @@ const ContinuousAnnotationTask: React.FC<Props> = (props) => {
   const videoPlayerOptions = useMemo(() => {
     let participant = ""
     let source = { ...props.spec.media[selectedCamViewIndex] }
+    // we let a plausible participant number to be replaced in the video url
+    // just so we don't get errors when things are loading.
+    let participant_substr_to_set_to_src_url: string = "1"
     if (validAnnotationsDataAndSelection) {
       // Note: consider optimizing such that if replaceable strings are not
       //       found, then it fallback into not having selectedAnnotationIndex
       //       as a dependency, and thus avoiding the videos from reload.
       participant = annotationsDataMirror[selectedAnnotationIndex].participant
-      const participantNumber = participant.replace("Participant_", "")
-      source.src = source.src.replace("{participant}", participantNumber)
+      participant_substr_to_set_to_src_url = participant.replace(
```

Comment:
What was wrong with participant number?

Reply:
Nothing, just making it more verbose to convey intent